### PR TITLE
Refactor FXIOS-4762 [v116] Update Pocket branding in settings

### PR DESCRIPTION
**Pull request by thatswinnie**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-4762)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/11599)

### Description
This is correcting the pocket branding for settings.

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Review by OrlaM — APPROVED** [review]

**Comment by thatswinnie** [comment]
@clarmso could you please review this PR? Thanks!

**Comment by clarmso** [comment]
> @clarmso could you please review this PR? Thanks!

👀 Looking! 👀

File: Tests/XCUITests/FxScreenGraph.swift [hunk 1]
```diff
@@ -695,7 +695,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
 
         screenState.gesture(forAction: Action.TogglePocketInNewTab) { userState in
             userState.pocketInNewTab = !userState.pocketInNewTab
-            app.tables.cells.switches["Recommended by Pocket"].tap()
+            app.tables.cells.switches["Thought-Provoking Stories"].tap()
```
Comment (clarmso) [inline Tests/XCUITests/FxScreenGraph.swift:698]:
```suggestion
            app.tables.cells.switches["Thought-Provoking Stories, Articles powered by Pocket"].tap()
```

**Comment by clarmso** [comment]
`testPocketEnabledByDefault()` is still failing on my end on both iPhone and iPad on this line. Tapping on the stories does not load the website. This test has been passing consistently on CI/CD.
https://github.com/mozilla-mobile/firefox-ios/blob/main/Tests/XCUITests/PocketTests.swift#L29

**Review by clarmso — APPROVED** [review]

**Pull request merged**